### PR TITLE
Update to audio-conferencing-smb

### PR DESCRIPTION
**Pull request by Exnav29**
In the Audio conferencing section corrected the wording that said shared numbers was covered earlier in the article.  When it actually discusses it in the next section afterwards.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/audio-conferencing-smb.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Your meeting organizers can now schedule meetings in Outlook. In Outlook, go to
 
 ## Audio Conferencing phone numbers
 
-There are two types of numbers that you can use for your conferencing bridge. You can use **shared numbers** (discussed earlier in this article) or **dedicated numbers**. Here's more information about each.
+There are two types of numbers that you can use for your conferencing bridge. You can use **shared numbers** (discussed in the next section of this article) or **dedicated numbers**. Here's more information about each.
```
[the remark this answers is not in the document]
Comment (JohanFreelancer9) [reply Teams/audio-conferencing-smb.md:84]: I would suggest removing this reference in totality because it does not really serve any purpose.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Hi @Exnav29 - thanks for your feedback. The steps in the Set up Audio Conferencing section uses shared numbers (which are automatically assigned). That is what "discussed earlier in this article" refers to. We'll update the article to clarify. Thanks!

**Pull request merged**